### PR TITLE
Extended the local job submit_job() function with optional args

### PR DESCRIPTION
**Pull request by alongd**
So it can be used external to ARC with optional arguments

### CODE REVIEW

File: arc/job/local.py [hunk 1]
```diff
@@ -219,19 +219,26 @@ def parse_running_jobs_ids(stdout: List[str],
...
+               submit_filename: Optional[str] = None,
                ) -> Tuple[Optional[str], Optional[str]]:
     """
     Submit a job.
 
     Args:
         path (str): The job's folder path, where the submit script is located (just the folder path, w/o the filename).
+        cluster_soft (str, optional): The server cluster software.
+        submit_cmd (str, optional): The submit command.
+        submit_filename (str, optional): The submit script dile name.
```
Comment (MichalKesl) [inline arc/job/local.py:233]: dile=file?

**Review by MichalKesl — COMMENTED** [review]
I think there is a typo.

File: arc/job/local.py (hunk 1 above)
Comment (alongd) [reply arc/job/local.py]: Thanks @MichalKesl, fixed

**Comment by alongd** [comment]
@MichalKesl, thanks - typo fixed,ready for approval (or additional comments)

**Review by MichalKesl — APPROVED** [review]

**Pull request merged**